moduleroot/.travis.yml.erb

```diff
@@ -106,11 +106,52 @@ matrix:
       <%= key %>: <%= job[key].gsub(/@@SET@@/, set['set']).gsub(/@@COLLECTION@@/, set.fetch('collection', 'puppet6')).gsub(/@@TESTMODE@@/, set.fetch('testmode', 'apply')) %>
 <%   end -%>
 <% end -%>
+<%#The following variables are used when use_litmus_travis is enabled.-%>
+<% 
+  if @configs['puppet_collection'].nil? 
+    @configs['puppet_collection'] = ['puppet5', 'puppet6'] 
+  end
+-%>
+<% 
+  if @configs['provision_list'].nil? 
+    @configs['provision_list'] = ['travis_el','travis_deb'] 
+  end
+-%>
+<%
+  if @configs['rvm'].nil? 
+    @configs['rvm'] = '2.5.3' 
+  end
+-%>
+<%
+  if @configs['run_tests'].nil? 
+    @configs['run_tests'] = 'parallel'
+  end
+-%>
+<% if @configs['use_litmus_travis'] -%>
+<%   @configs['puppet_collection'].each do |puppet_version|-%>
```

Comment:
You don't need to keep turning the interpolation on-and-off per line.  It's quite ok to do:

```
<%
 if @configs['use_litmus_travis']
   @configs['puppet_collection'].each do |puppet_version|
   @configs['provision_list'].each do |platform|-%>
```

Reply:
Makes it much easier to read.